app/routes/task_routes.py

```diff
@@ -0,0 +1,113 @@
+from flask import Blueprint, jsonify, make_response, request, abort
+from flask.helpers import make_response
+from flask.json import tojson_filter
+from flask.signals import request_tearing_down
+from werkzeug.utils import header_property
+from app.models.task import Task
+from app import db
+from datetime import datetime
+from dotenv import load_dotenv
+import requests, os 
+from app.routes.utils import valid_int
+
+load_dotenv()
+
+task_bp = Blueprint("task", __name__,url_prefix ="/tasks")
+
+# Helper Functions
+def get_task_from_id(task_id):
+    valid_int(task_id, "task_id")
+    return Task.query.get_or_404(task_id, description="{task not found}")
+
+def post_slack_message(message):
+    token = os.environ.get('SLACK_TOKEN')
+    CHANNEL_ID = "C02KD4B5A07"
+
+    Headers = {"Authorization": "Bearer xoxb-" + token}
```

Comment:
This line causes a failure if the `SLACK_TOKEN` doesn't exist in the environment (which is why the automated tests fail).  

It's perfectly reasonable to assume you will have a key (and the tests pass if given the token of `token` so would work if the token was expired) this was just a quirk of our Learn setup.

If you wanted to fix the test failures you could use interpolation to do this instead:
```suggestion
    Headers = {"Authorization": f"Bearer xoxb-{token}"}
```

Again, this isn't a bug, just a quirk of our Learn setup.

It's just the kind of thing where _I_ would want to know why it failed on Learn and not on my local machine.